exec/query.R

```diff
@@ -23,6 +23,7 @@ OPTIONS:
   --packages       Run the packages query.
```

Comment:
I would recommend being more descriptive here. What exactly is the user getting? So, id say something like:

"Returns a 3 column table (filepath, classpath, file_content), where every row is defined by a class docstring" or 

"Returns a 3 column table (filepath, classpath, file_content), where every row is defined by a class name"

etc. Remember the user will have no idea what "imports query" means.